{"name": "登录测试用例", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at AndroidUiautomator2Driver.findElOrEls (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-android-driver\\lib\\commands\\find.js:75:11)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\commands\\find.js:33:12)\n    at AndroidUiautomator2Driver.findElement (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\commands\\find.js:53:10)", "trace": "self = <cases.app_ui.android.test_login.TestLogin object at 0x000001EF29F06B50>, case = {'description': '账号错误', 'expected': '账号或密码错误', 'password': 'w123456789', 'username': '000000000@qq.com'}\n\n    @allure.feature(\"登录模块\")\n    @allure.title(\"登录测试用例\")\n    @pytest.mark.parametrize(\"case\", case_data[\"login_cases\"])\n    def test_login(self, case):\n        \"\"\"\n        登录测试用例,\n        \"\"\"\n        INFO.logger.info(case)\n    \n        # 勾选隐私协议\n>       self.lg.select_agreement()\n\ncases\\app_ui\\android\\test_login.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npage_objects\\app_ui\\android\\pages\\login_page.py:21: in select_agreement\n    if self.is_checked(*le.AGREEMENT):\ncommon\\appium\\appOperator.py:164: in is_checked\n    element = self.locator(name, text)\ncommon\\appium\\appOperator.py:58: in locator\n    element = self.wait.until(ec.visibility_of_element_located((name, text)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c28ada0d-8e20-4f16-955e-b54d8acf462a\")>, method = <function visibility_of_element_located.<locals>._predicate at 0x000001EF2A232160>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except InvalidSelectorException as e:\n                raise e\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       NoSuchElementError: An element could not be located on the page using the given search parameters.\nE           at AndroidUiautomator2Driver.findElOrEls (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-android-driver\\lib\\commands\\find.js:75:11)\nE           at runMicrotasks (<anonymous>)\nE           at processTicksAndRejections (internal/process/task_queues.js:97:5)\nE           at AndroidUiautomator2Driver.findElOrElsWithProcessing (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\commands\\find.js:33:12)\nE           at AndroidUiautomator2Driver.findElement (C:\\Program Files\\Appium Server GUI\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\commands\\find.js:53:10)\n\nC:\\Users\\w8345\\.conda\\envs\\pytestAutoApi\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:89: TimeoutException"}, "description": "\n        登录测试用例,\n        ", "attachments": [{"name": "log", "source": "c5abe668-f733-476b-806a-01eaf8ee89f1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'username': '000000000@qq.com', 'password': 'w123456789', 'expected': '账号或密码错误', 'description': '账号错误'}"}], "start": 1757641783526, "stop": 1757641793756, "uuid": "8cf57e1c-8a6b-4236-8621-76cc9104432a", "historyId": "9e5b6a4a847c60234bd093c2ff861a4d", "testCaseId": "9b772e3c4d74e39c94b5c8b74ce1a0be", "fullName": "cases.app_ui.android.test_login.TestLogin#test_login", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "cases.app_ui.android"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "wk"}, {"name": "thread", "value": "27980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.app_ui.android.test_login"}]}